lib/ledger_sync/adaptors/quickbooks_online/adaptor.rb

```diff
@@ -101,14 +101,15 @@ def refresh!
         end
 
         def revoke_token!
-          request(
-            :post,
-            REVOKE_TOKEN_URI,
+          headers = OAUTH_HEADERS.dup.merge('Authorization' => 'Basic ' + Base64.strict_encode64(client_id + ':' + client_secret))
+          LedgerSync::Adaptors::Request.new(
             body: {
               token: access_token
-            }.to_json,
-            headers: OAUTH_HEADERS.dup
-          ).status == 200
+            },
+            headers: headers,
+            method: :post,
+            url: REVOKE_TOKEN_URI
+          ).perform.status == 200
```

Comment:
I think it probably best to return the request here for debugging purposes actually.  But I'll go ahead and approve and we can do that downstream if we want.